src/main/java/com/snowflake/kafka/connector/internal/streaming/StreamingUtils.java

```diff
@@ -73,6 +75,16 @@ public class StreamingUtils {
   public static final String STREAMING_CONSTANT_OAUTH_CLIENT_SECRET = "oauth_client_secret";
   public static final String STREAMING_CONSTANT_OAUTH_REFRESH_TOKEN = "oauth_refresh_token";
 
+  // contains config properties that should not be exposed to the customer as they may contain PII
+  // or authentication information
+  public static final List<String> SENSITIVE_STREAMING_CONFIG_PROPERTIES =
```

Comment:
This looks risky, it's possible that people forgot updating this list and log sensitive data, should we do the other way around? Only log the information that is in the list

Reply:
Good point, inverted it to LOGGABLE_STREAMING_CONFIG_PROPERTIES